frontend/src/components/pages/GlobalCodeReviewDashboard/Presentation.jsx

```diff
@@ -127,6 +137,64 @@ export default function Presentation({
     return true;
   }
 
+  const initialPullRequestOrderFilters = [
```

Comment:
This is good stuff. It would be better if it was in the index file.

I realize that I did take some shortcuts when making the original version of this file - sometimes it's the best way to unblock people (the scrum masters were really suffering with the code review wrangling). I left some technical debt.

But the standard is still to separate things out - smarts in index.jsx and looks in presentation.jsx